index.html

```diff
@@ -1,11 +1,582 @@
+      </div>
+
+      <nav id="nav-menu-container">
+        <ul class="nav-menu">
+          <li class="menu-active"><a href="index.html">Home</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#event">Essay</a></li>
+          <li><a href="#timeline">Timeline</a></li>
+          <li><a href="#faq">FAQ</a></li>
+          <!-- <li><a href="#supporters">Sponsors</a></li> -->
+          <li><a href="mailto:abhi24@mit.edu">Contact</a></li>
+          <li class="buy-tickets"><a href="#register">Register Now</a></li>
+        </ul>
+      </nav><!-- #nav-menu-container -->
+    </div>
+  </header><!-- End Header -->
+
+  <!-- ======= Intro Section ======= -->
+  <section id="intro">
+    <div class="intro-container" data-aos="zoom-in" data-aos-delay="100">
+      <h1 class="mb-4 pb-0"><img src="assets/img/logo.png" class="logo" alt="" title=""></h1>
+      <p class="mb-4 pb-0">Virtual Meetup for High School Students<br>Network, Interact, and Write Papers<br><small style="font-weight: 900;">November 28th to December 7th</small></p>
+      <!-- <a href="https://www.youtube.com/watch?v=jDDaplaOz7Q" class="venobox play-btn mb-4" data-vbtype="video" data-autoplay="true"></a> -->
+      <div style="display:inline-block"><a href="#about" class="about-btn scrollto">Learn More</a>
+      <a href="#register" class="register-btn scrollto">Register</a></div>
+      <br>
+      <br>
+      <!-- <p>Powered by:</p> -->
+      <div style="display:inline-block">
+        <img class="powered" src="assets/img/mit.png" />
+        <!-- <img class="powered" src="assets/img/pathcheck.png" /> -->
+      </div>
+    </div>
+  </section><!-- End Intro Section -->
+
+  <main id="main">
+
+    <!-- ======= About Section ======= -->
+    <section id="about">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-12" style="margin-bottom:30px">
+            <h2>MIT Pathfinders is a joint event with the MIT Vaccine Conference that permits high school students to interact with each other to learn and write about vaccination as it relates to the coronavirus pandemic.</h2>
+          </div>
+          <div class="col-lg-3">
+            <h3>What</h3>
+            <p>A virtual meetup to support the digital pandemic response where students can learn more about vaccination and vital information regarding the pandemic! <!--Learn more about the format <a href="format.html">here</a>.</a>--></p>
+          </div>
+          <div class="col-lg-3">
+            <h3>Who</h3>
+            <p>High school students aged 13+</p>
+          </div>
+          <div class="col-lg-3">
+            <h3>Where</h3>
+            <p>100% Virtual on the <a href="https://join.slack.com/t/mitpathfinders/shared_invite/zt-iiulq7o6-ANIDzqgvvsJlySqBVqoVmQ">MIT Pathfinders Slack</a></p>
+          </div>
+          <div class="col-lg-3">
+            <h3>When</h3>
+            <p>November 28th to December 7th</p>
```

Comment:
just date of event itself. you can add the submission deadlines on a newline if you want:

**Abstract Submission Deadline:** Nov 1
**Full Paper Submission Deadline:** Nov 1
**MIT Pathfinders Presentation:** Nov 1
etc